invenio_rdm_records/records/systemfields/access/grants.py

```diff
@@ -130,7 +130,7 @@ def subject_id(self):
         if self._subject_id:
             return self._subject_id
 
-        if self.subject_type == "role":
+        if self.subject_type == "group":
```

Comment:
minor: maybe the rename should be also discussed with zenodo team, not sure if there are consequences of changing the name here